crates/core/src/commands/check.rs

```diff
@@ -37,7 +38,41 @@ pub struct CheckOptions {
     pub read_data: bool,
 }
 
-impl CheckOptions {
+#[allow(clippy::enum_glob_use)]
+use CheckError::*;
```

Comment:
I think glob imports are bad because they hinder readability and confuse. They are usually being frowned upon, so I think they should be removed here as well, and just the normal enum syntax should be used. I removed it all over the crate, when I factored out the library actually.

Reply:
It's also hard to review online, by the way. So I didn't go further still, will do it after the change, so I know where the enum is being used. 😬

Reply:
Ok, I removed the enum glob import...